src/server/req_signal.c

```diff
@@ -548,6 +552,12 @@ create_resreleased(job *pjob)
 			free(resreleased);
 			return 1;
 		}
+		/* If there are no resources released on this vnode then add a dummy "ncpus=0"
+		 * This is needed otherwise scheduler will not be able to assign this chunk to
+		 * the job while trying to resume it
+		 */
+		if (no_res_rel)
+			strcat(resreleased,dflt_ncpus_rel);
```

Comment:
I think it will not happen. Even if we submit a job without requesting any resource exec_vnode does contain single ncpu requested on each chunk. There was an allocation problem although where we did not account for null character. I've rectified that.